backend/internal/service/oidc_service.go

```diff
@@ -1957,6 +2059,80 @@ func (s *OidcService) downloadAndSaveLogoFromURL(parentCtx context.Context, tx *
 	return nil
 }
 
+func (s *OidcService) downloadAndSaveDarkLogoFromURL(parentCtx context.Context, tx *gorm.DB, clientID string, raw string) error {
```

Comment:
Please create a single function for light and dark, with a parameter like `dark bool` to switch between the two. There's a lot of duplicate code here